sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/APISpec.groovy

```diff
@@ -655,6 +656,10 @@ class APISpec extends StorageSpec {
         }
     }
 
+    def compareDatesWithPrecision(OffsetDateTime expectedTime, OffsetDateTime actualTime) {
+        return expectedTime.truncatedTo(ChronoUnit.MICROS) == actualTime.truncatedTo(ChronoUnit.MICROS)
```

Comment:
What is `ChronoUnit.MICRO`?

Reply:
It would be good to make precision a parameter of this method, so it can be unique in different tests.